docs/content/docs/operators/feature/vectorassembler.md

```diff
@@ -44,11 +46,12 @@ Types of input columns must be either vector or numerical value.
 
 ### Parameters
 
-| Key           | Default    | Type     | Required | Description                                                                    |
-|---------------|------------|----------|----------|--------------------------------------------------------------------------------|
-| inputCols     | `null`     | String[] | yes      | Input column names.                                                            |
-| outputCol     | `"output"` | String   | no       | Output column name.                                                            |
-| handleInvalid | `"error"`  | String   | no       | Strategy to handle invalid entries. Supported values: 'error', 'skip', 'keep'. |
+| Key             | Default    | Type      | Required | Description                                                                    |
+|-----------------|------------|-----------|----------|--------------------------------------------------------------------------------|
+| inputCols       | `null`     | String[]  | yes      | Input column names.                                                            |
+| outputCol       | `"output"` | String    | no       | Output column name.                                                            |
+| inputSizes      | `null`     | Integer[] | yes      | Sizes of the assembling elements.                                              |
```

Comment:
The type of this parameter should be `int[]` instead of `Integer[]`.

Reply:
I think the integer is much better.